Cargo.toml

```diff
@@ -10,9 +10,10 @@ async-trait = "0.1"
 clap = { version = "4.4", features = ["derive", "env"] }
```

Comment:
Just need a version bump here and types tag changed. I'll re-approve (if needed) when the conflict fixes.

Reply:
Running CI now.